server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -649,7 +649,10 @@ public void apply(Settings value, Settings current, Settings previous) {
                 RemoteRefreshSegmentPressureSettings.MIN_CONSECUTIVE_FAILURES_LIMIT,
                 RemoteRefreshSegmentPressureSettings.UPLOAD_BYTES_MOVING_AVERAGE_WINDOW_SIZE,
                 RemoteRefreshSegmentPressureSettings.UPLOAD_BYTES_PER_SEC_MOVING_AVERAGE_WINDOW_SIZE,
-                RemoteRefreshSegmentPressureSettings.UPLOAD_TIME_MOVING_AVERAGE_WINDOW_SIZE
+                RemoteRefreshSegmentPressureSettings.UPLOAD_TIME_MOVING_AVERAGE_WINDOW_SIZE,
+
+                // Settings related to node join
+                JoinHelper.CLUSTER_MANAGER_VALIDATE_JOIN_CACHE_INTERVAL
```

Comment:
[minor]: there is another JoinHelper setting above L565, keep it along with that

Reply:
Ack.

Reply:
wondering why would you want to keep it time based, wouldn't you want to invalidate this on a newer version? How does time help? Is this about bounded consistency?

Reply:
@Bukhtawar  explained in detail on this thread https://github.com/opensearch-project/OpenSearch/pull/7321#discussion_r1200219753

Reply:
Update : removing the setting.